### DISCUSSION TITLE
Which version of boost is used? Version 1.0?

### DISCUSSION THREAD

**Discussion by craigtao** [artyom-beilis/cppcms#95] [Announcements]
Which version of boost is used? Version 1.0?

thanks.

**Comment by artyom-beilis** [comment]
cppcms does not use boost at all.

It has internal "booster" that has some similarities and some of the code taken from boost but it is build for stable api and it is part of standard CppCMS build. And starting from 2.0 many if its features replaced with C++11 libraries.

**Comment by craigtao** [comment]
wow! amazing! you are a master.

CppCMS is very interesting because it converts tmpl files containing HTML layouts into cpp files through python scripts: statically compile HTML into native apps

**Comment by craigtao** [comment]
CppCMS is similar to the .NET platform's webform
